pkgs/by-name/ut/ut/package.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, ninja
+}:
+stdenv.mkDerivation rec {
+  pname = "ut";
+  version = "2.0.0";
+
+  cmakeFlags = [
+    "-DBOOST_UT_ALLOW_CPM_USE=OFF"
+  ];
+
+  src = fetchFromGitHub {
+    owner = "boost-ext";
+    repo = "ut";
+    rev = "v${version}";
```

Comment:
Use the [`finalAttrs` idiom](https://nixos.org/manual/nixpkgs/unstable/#mkderivation-recursive-attributes).

Reply:
Done.